src/content/docs/apm/agents/php-agent/other-integrations/magento-integration.mdx

```diff
@@ -0,0 +1,281 @@
+---
+title: Magento monitoring integration
+tags:
+    - New Relic integrations
+    - Magento
+metaDescription: Monitor your Magento application.
+---
+
+import apmMagentoDashboardOverview from 'images/apm_screenshot-full_magento-dashboard-overview.png'
+
+Our Magento integration monitors the performance of your Magento ecommerce application, helping you diagnose issues in your app's performance and respond by optimizing your code. Our Magento integration relies on our agents to collect and report your app data, and
+you'll have access to a pre-built dashboard with important app metrics, like errors, application crashes, app launches, app installations, HTTP errors, and network failures.
+
+<img
+    title="Magento dashboard overview"
+    alt="screenshot of Magento dashboard"
+    src={apmMagentoDashboardOverview}
+/>
+
+<figcaption>
+    After setting up our Magento integration, we give you a dashboard for your Magento app metrics.
+</figcaption>
+
+<Steps>
+    <Step>
+        ## (Optional) Install our infrastructure agent
+
+        Before getting Magento data into New Relic, first install our infrastructure agent, then you will be able to get details about your host.
+        * Follow our guided install to instrument your system with the infrastructure agent. Or if you prefer, you can install the infrastructure agent by following the docs.
+        * By integrating this infrastructure agent in your system, one can able to 
+        know the CPU usage,Memory usage,Storage usage,Disk usage and Network traffic etc.
+
+        </Step>
+
+    <Step>
+        ## Install our PHP agent
+
+        Install PHP agent to instrument your Magento application and get Application and Browser related Metrics like AJAX performance,Error Rate,Duration,Core web vitals and Throughput.
+        Follow php documentation: https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements/ to learn more about.
+
+        From one.newrelic.com:
+
+        1. Click Add data
+        2. Click Application monitoring
+        3. Click on PHP card
+        4. Select an account
+        5. Click Begin installation
+        6. Click on On host
+        7. Copy the command and run it on your terminal
+
+        </Step>
+    
+    <Step>
+        ## Install our MySQL agent
```

Comment:
Since MySQL also requires the infra agent, should this step be optional (if the infra agent is optional)?

Reply:
Infra agent is not optional